host/image.h

```diff
@@ -32,13 +32,13 @@
 #ifndef _IMAGE_H_
 #define _IMAGE_H_
 
-#include <setjmp.h>
+#include <efisetjmp.h>
 #include <ewdrv.h>
 
 typedef struct image {
 	EFI_LOADED_IMAGE prot;
 	void *data;
-	jmp_buf jmp;
+	jmp_buf jmp[1];
```

Comment:
Actually to make the change smaller, setjmp() and longjmp() function wrapper from gnuefi lib need a pointer to jmp_buf as parameter.
If you really dont like it, I can do the change on the callers.